### ISSUE TITLE
Direct Localization in Underwater Acoustics via Convolutional Neural Networks: A Data-Driven Approach. (arXiv:2207.10222v1 [cs.LG])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#21281]
Direct Localization in Underwater Acoustics via Convolutional Neural Networks: A Data-Driven Approach. (arXiv:2207.10222v1 [cs.LG])<br>
https://ift.tt/4PXxej2<br>
<p>Direct localization (DLOC) methods, which use the observed data to localize a source at an unknown position in a one-step procedure, generally outperform their indirect two-step counterparts (e.g., using time-difference of arrivals). However, underwater acoustic DLOC methods require prior knowledge of the environment, and are computationally costly, hence slow. We propose, what is to the best of our knowledge, the first data-driven DLOC method. Inspired by classical and contemporary optimal model-based DLOC solutions, and leveraging the capabilities of convolutional neural networks (CNNs), we devise a holistic CNN-based solution. Our method includes a specifically-tailored input structure, architecture, loss function, and a progressive training procedure, which are of independent interest in the broader context of machine learning. We demonstrate that our method outperforms attractive alternatives, and asymptotically matches the performance of an oracle optimal model-based solution.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
